.nix/config.nix

```diff
@@ -55,6 +55,7 @@
     gen "8.17" "mathcomp-2.1.0" //
     gen "8.18" "mathcomp-2.1.0" //
     gen "8.19" "mathcomp-2.2.0" //
+    { "coqmaster+mcmaster".ocamlPackages = { elpi.override.version = "1.19.2"; }; } //
     gen "master" "master";
```

Comment:
Maybe swap the two lines, just in case an override in the second wipes out the first one?

Reply:
Swapping these two lines (and then `genNixActions`) rather seems to override everything (see the last commit).

Reply:
Also it does not seem to fix the issue.

Reply:
Indeed, this seems to work:
```
    gen = coqv: mcv: elpiv:
      { "coq${coqv}+mc${mcv}" = {
	coqPackages = {
               coq.override.version = coqv;
               mathcomp.override.version = mcv;
               mathcomp.job = false;
        } // (if (coqv == "master") then {
            coq-elpi.override.version = "master";
            hierarchy-builder.override.version = "master";
	} else {}) // {
            mathcomp-real-closed.override.version = "master";
            mathcomp-bigenough.override.version = "1.0.1";
        };
        ocamlPackages = if (elpiv != "") then { elpi.override.version = elpiv; } else {};
      }; }; in
    gen "8.16" "mathcomp-2.1.0" "" //
    gen "8.17" "mathcomp-2.1.0" "" //
    gen "8.18" "mathcomp-2.1.0" "" //
    gen "8.19" "mathcomp-2.2.0" "" //
    gen "master" "master" "1.19.2";
```

Reply:
Thanks!